code-challenges.test.js

```diff
@@ -11,49 +11,198 @@
 // Run the file with the following command: $ yarn jest
 
 
+//Ran jest and it failed as planned
+
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// //I want to write a fuction that contains a conditional statement to meet the condition if an input number is divisible by 3
+// const divisibleByThree = (num) => {
+//   //This will define the functiion annd give it an input
+//   if (num % 3 === 0){
+//     //The first condition checks to see if the number is divisible by 3 using the modulo operator. If an input number is divided by 3 and has no remainder it is divisible by three. 
+//     return `${num} is divisible by three`
+//     //This statement will log to the console if the first condition met evaluates to true.
+//   } else {
+//     //This else statement catches all other inputs that are not numbers divisible by 3.
+//     return `${num} is not divisible by three`
+//   }//The return statement if the first condition is not met.
+// }
+
+
+//Refactored code moved above line 39 to make test pass. Function must be defined in order to pass test. See lines 80-87 for refactored code with psuedo code. 
+const divisibleByThree = (num) => {
+  if (num % 3 === 0) {
+    return `${num} is divisible by three`;
+  }
+  return `${num} is not divisible by three`
+}
+
 var num1 = 15
+
+describe("divisibleByThree", () => {
+  it("returns a string of `15 is divisible by three` if a number is divisible by 3 or not based on a number input", () => {
+    expect(divisibleByThree(num1)).toEqual(`15 is divisible by three`)
+  })
+})
+
+console.log(divisibleByThree(num1))
 // Expected output: "15 is divisible by three"
+
+
+
 var num2 = 0
+console.log(divisibleByThree(num2))
 // Expected output: "0 is divisible by three"
+
+describe("divisibleByThree", () => {
+  it("returns a string of `0 is divisible by three` if a number is divisible by 3 or not based on a number input", () => {
+    expect(divisibleByThree(num2)).toEqual(`0 is divisible by three`)
+  })
+})
+
 var num3 = -7
+console.log(divisibleByThree(num3))
 // Expected output: "-7 is not divisible by three"
 
+describe("divisibleByThree", () => {
+  it("returns a string of `-7 is not divisible by three` if a number is divisible by 3 or not based on a number input", () => {
+    expect(divisibleByThree(num3)).toEqual(`-7 is not divisible by three`)
+  })
+})
```

Comment:
this is good but I am gonna leave a standard test using what you wrote here for your ref: 

```javascript
describe("divisibleByThree", () => {
  it("returns a string of `15 is divisible by three` if a number is divisible by 3 or not based on a number input", () => {
    expect(divisibleByThree(num1)).toEqual(`15 is divisible by three`)
    expect(divisibleByThree(num2)).toEqual(`0 is divisible by three`)
    expect(divisibleByThree(num3)).toEqual(`-7 is not divisible by three`)
  })
})
```